### DISCUSSION TITLE
Put AI computation inside 3d space, GPU computation is abstracted out.

### DISCUSSION THREAD

**Discussion by levirtevs** [explosion/spaCy#11445] [New Features & Project Ideas]
**Ideas**

1. write a truth finder program - Too simple to put in words, too complex for next step. (A solution put in words is half done). (instructions, it gets information dump from monitor not as file, learns to read it, extracts facts, converts facts to rules, uses rules to rebuild truth in 3d space. Goal is that, rules cross each other out, at the core something beautiful can be found.) Tried in blender path, I built it from source and gave up be cause it is too much for head. So got rid of light, sound, space and sticking to words as single source of truth. Words create sound, sound creates light.
2. why not use nlp to extract facts - Currently I'm in this path. I have applied for a job at explosion, however I'm not sure if I head in that direction because job responsibilities will hinder me from chasing my interest.
3. Why not use nlp to make an OS ?
4. Why not use nlp to build tooling ?
5. Why not use nlp to mimic our world inside computer?
6. Keep continuing the questions, you will find a pattern.

**Where I need help**
The way I visualise, I see at the end of nlp facts, it needs to be broken down into computer command.  have come this far. This idea is stuck in my mind, unlike other ideas, this keeps hitting back.

**Comment by adrianeboyd** [comment]
Sorry, this discussion forum is focused on spaCy and isn't the right place for this kind of topic.
